src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -886,7 +886,7 @@ private Bid toBid(TargetingBidInfo targetingBidInfo,
         final ExtBidPrebid extBidPrebid = ExtBidPrebid.builder()
                 .bidid(bidInfo.getGeneratedBidId())
                 .type(bidType)
-                .targeting(targetingKeywords)
+                .targeting(MapUtils.isEmpty(targetingKeywords) ? null : targetingKeywords)
```

Comment:
I'm not sure why {{null}} is a better return value than an empty Map :sweat_smile: 
Will the {{targeting}} object be removed from the resulting json?

Reply:
Yep